x/permission/types/common.pb.go

```diff
@@ -44,14 +44,14 @@ const (
 	ACTION_LIST_OBJECT         ActionType = 8
 	ACTION_UPDATE_GROUP_MEMBER ActionType = 9
 	ACTION_DELETE_GROUP        ActionType = 10
-	ACTION_UPDATE_OBJECT_INFO  ActionType = 11
-	ACTION_UPDATE_GROUP_EXTRA  ActionType = 12
+	ACTION_UPDATE_OBJECT       ActionType = 11
+	ACTION_UPDATE_GROUP_META   ActionType = 12
 	ACTION_TYPE_ALL            ActionType = 99
 )
 
 var ActionType_name = map[int32]string{
 	0:  "ACTION_UNSPECIFIED",
-	1:  "ACTION_UPDATE_BUCKET_INFO",
+	1:  "ACTION_UPDATE_BUCKET",
```

Comment:
Do our SDK, JS and CMDLINE need to also update?
I mean if they do not update `greenfield` dependency, do they still work?